test/fixtures/files/json/isa_templates/template_attributes_schema.json

```diff
@@ -0,0 +1,112 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema",
```

Comment:
I don't think this belongs with the test fixtures, as the schema is used in rake tasks. It could go together with the task